src/cli.ts

```diff
@@ -81,6 +96,11 @@ Boolean values can be set to false using the 'no-' prefix.
       Prettier configuration
   --unreachableDefinitions
       Generates code for definitions that aren't referenced by the schema
+  --namePriority={${values(CliNameSource).join(',')}},...
+      A commma seperated list of priorties for type naming.
```

Comment:
Typo/Nit-picking: "comma-separated", "priorities"

Reply:
thanks, I fixed it.